source/transfer_list.rst

```diff
@@ -447,9 +447,9 @@ The following entry types are currently defined:
 - single HOB block entry: tag_id = 2 (:numref:`hob_block_entry`).
 - HOB list entry: tag_id = 3 (:numref:`hob_list_entry`).
 - ACPI table aggregate entry: tag_id = 4 (:numref:`acpi_aggr_entry`).
-- OPTEE pagable part entry: tag_id = 5 (:numref:`optee_pagable_part_entry`).
-- TOS FW config entry: tag_id = 6 (:numref:`tos_fw_config_entry`).
-- OPTEED AARCH entry: tag_id = 7 (:numref:`opteed_aarch_entry`).
+- OPTEE pagable part entry: tag_id = 0x100 (:numref:`optee_pagable_part_entry`).
```

Comment:
I don' t think we need 256 entries for OP-TEE. I suggest using 0x10 or 0x20

Reply:
How about to start from 0x20? Just want to reserve more spaceholders for the "regular" ones.